main.py

```diff
@@ -252,6 +252,32 @@ def list(self, dpid=None):
 
         return jsonify(switch_flows)
 
+    @listen_to('kytos.flow_manager.flows.(install|delete)')
+    def event_add_flow(self, event):
+        """Install or delete flows in the switches through events.
+
+        Install or delete Flow of switches identified by dpid.
+        """
+        try:
+            dpid = event.content['dpid']
+            flow_dict = event.content['flow_dict']
+        except KeyError as error:
+            log.error("Error getting fields to install or remove "
+                      f"Flows: {error}")
+            return
+
+        if event.name == 'kytos.flow_manager.flows.install':
+            command = 'add'
+        else:
+            command = 'delete'
```

Comment:
Having the `delete` command as a default may lead to hard-to-find bugs in the future. I'd prefer to make it explicit:

```suggestion
        elif event.name == 'kytos.flow_manager.flows.delete':
            command = 'delete'
        else:
            msg = f'Invalid event "{event.name}", should be install|delete'
            raise ValueError(msg)
```